.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--
+  Thanks for participating in this project!
+  Please:
+    - do at least one GitHub search in current issues, maybe your question is already here
+-->
+
+## 📚 Documentation
```

Comment:
```suggestion
## Description
```

Did you mean "Description"? I don't understand "Documentation" as a title.

Reply:
I did mean Documentation as this repo is mostly about Documentation

Reply:
Then I don't think it makes sense as an issue title, it doesn't describe what users will have to fill.

Reply:
I have updated the template.